src/main/java/at/hannibal2/skyhanni/config/features/GUIConfig.java

```diff
@@ -133,6 +133,15 @@ public static class InGameDateConfig {
         @FeatureToggle
         public boolean includeSunMoon = true;
 
+        @Expose
+        @ConfigOption(
+            name = "Show Date Ordinal",
+            desc = "Show the date's ordinal suffix. Ex: (1st <-> 1, 22nd <-> 22, 23rd <-> 3, 24th <-> 24, etc.)"
+        )
+        @ConfigEditorBoolean
+        @FeatureToggle
```

Comment:
This is no standalone feature, hence no feature toggle.

Reply:
> This is no standalone feature, hence no feature toggle.
pardon, pls elaborate